src/pages/ProtocolPage.jsx

```diff
@@ -135,7 +138,7 @@ function ProtocolPage(props) {
                     <div className="row d-flex align-items-center justify-content-center h-100 p-0 m-0">
                         <div className="col col-md-10 d-flex flex-column h-100 p-4 px-lg-5">
                             <div className="d-flex flex-column flex-grow-1">
-                                <div className="row justify-content-center g-2 mb-4">
+                                <div className="row justify-content-center g-2 m-0 pb-4">
```

Comment:
```suggestion
                                <div className="row justify-content-center g-2 pb-4">
```
- Convém não definir margem de row, pois o componente faz isso dinamicamente para comportar os gutters;